config/constants.go

```diff
@@ -110,4 +110,7 @@ const (
 	// WorkspaceVolumeName contains the name of the volume that is shared between
 	// the init containers and containers for a driver or worker pod.
 	WorkspaceVolumeName = "workspace"
+
+	// KillAfter sets the allowed response time for a pod after timeout.
+	KillAfter = "KILL_AFTER"
```

Comment:
nit: name this `KillAfterEnv` to match the style of already-existing `ScenariosFileEnv`?